cmd/kubermatic-installer/cmd_mirror_images.go

```diff
@@ -85,6 +89,12 @@ func MirrorImagesCommand(logger *logrus.Logger, versions kubermaticversion.Versi
 			if len(args) >= 1 {
 				opt.Registry = args[0]
 			}
+
+			if strings.Contains(opt.Registry, "local://") {
+				opt.Archive = true
+				opt.ArchivePath = opt.Registry[len("local://"):]
```

Comment:
let's use `strings.TrimPrefix` here so we don't run into weird out-of-bounds problems (again, shouldn't happen ... but you never know).